src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const arr = [];
+
+  for (const key in robot) {
+    if (arr.includes(robot[key])) {
```

Comment:
You can use hasOwnProperty in this case to check if robot value repeat

Reply:
Fixed. I forgot about this method when I was writing code))